execution_engine/src/core/execution/executor.rs

```diff
@@ -434,6 +438,7 @@ impl Executor {
                 }
             }
             DirectSystemContractCall::GetEraValidators => {
+                // TODO See if these panics can be removed.
```

Comment:
TODO in this PR or in a subsequent one?

Reply:
Subsequent PR

Reply:
Well spotted, but we're aiming for a subsequent one.